ide/juvix-examples/Id-Strings.ju

```diff
@@ -0,0 +1,16 @@
+let string = %Michelson.string
```

Comment:
I think we don't need this file here? There are multiple examples in `test/examples`.